examples/with-gnosis/src/createNewEthereumPrivateKey.ts

```diff
@@ -14,12 +18,37 @@ export async function createNewEthereumPrivateKey() {
     "`process.env.PRIVATE_KEY_ID` not found; creating a new Ethereum private key on Turnkey...\n"
   );
 
-  const privateKeyName = `ETH Key ${String(
-    Math.floor(Math.random() * 10000)
-  ).padStart(4, "0")}`;
+  // Use `withAsyncPolling` to handle async activity polling.
+  // In this example, it polls every 250ms until the activity reaches a terminal state.
+  const mutation = withAsyncPolling({
+    request: PublicApiService.postCreatePrivateKeys,
+    refreshIntervalMs: 250,
```

Comment:
I think one of these examples should omit the `refreshIntervalMs`, to show that this doesn't have to be specified.